cg/meta/report/api.py

```diff
@@ -68,12 +70,12 @@ def _handle_missing_report_data(self, accept_missing_data, delivery_data, case_i
             )
 
     def create_delivery_report_file(
-        self, case_id: str, file_path: Path, accept_missing_data: bool = False
+        self, case_id: str, file_path: Path, analysis_date, accept_missing_data: bool = False
```

Comment:
Again I think we should use type hints, especially when they are used in the same function. And add return types, if nothing is returned we can use 

```python
def foo(bar: int) -> None:
    pass
```

Reply:
typehint added